src/angularComponent/ngcOmniboxComponent.js

```diff
@@ -0,0 +1,6 @@
+import NgcOmniboxController from './ngcOmniboxController.js';
```

Comment:
Should we at least have an empty template file in this commit as well? I'd like there to be something that verifiably works. i.e. if I `jspm install` this today, and include it as a dependency of my angular module, and then I do `<ngc-omnibox>` in my template, I should see something, maybe a "hello world" or something.